LibGAP.jl/src/conversion.jl

```diff
@@ -39,11 +40,12 @@ end
 
 from_gap(x,::Type{Any})    = x
 from_gap(x,::Type{String}) = from_gap_string( x )
+from_gap(x,::Type{Symbol}) = from_gap_symbol( x )
```

Comment:
Is there a deeper reason for having all these duplicate functions? I.e., any reason to have `from_gap_symbol`, doesn't `from_gap(x,::Type{Symbol})` suffice?